cylc/flow/scripts/lint.py

```diff
@@ -443,9 +442,9 @@ def get_option_parser() -> COP:
         dest='linter'
     )
     parser.add_option(
-        '--reference', '--ref', '-R',
+        '--list-codes',
         help=(
-            'Print Reference for error codes.'
+            'List all linter codes.'
```

Comment:
Having both `-r` and `-R` as options seems confusing. I don't think we need a short option for a documentation command. How about `--list-codes`?

Reply:
Legit - I've confused myself at least once.